src/centralserver/admin-ui/package.json

```diff
@@ -5,14 +5,15 @@
   "scripts": {
     "serve": "rm -rf src/openapi-types && npm run generate-types && vue-cli-service serve",
     "build": "rm -rf src/openapi-types && npm run generate-types && vue-cli-service build",
-    "generate-types": "openapi --input ../admin-service/src/main/resources/openapi-definition.yaml --output ./src/openapi-types --exportCore true --client fetch",
+    "generate-types": "npm run prepare-api-definition && openapi --input ./openapi-definition-frontend.yaml --output ./src/openapi-types/cs-types --exportCore true --client fetch && openapi --input ../../common-rest-api/src/main/resources/common-openapi-definition.yaml --output ./src/openapi-types/common-types --exportCore true --client fetch",
```

Comment:
This branch https://github.com/nordic-institute/X-Road/blob/XRDDEV-1535-openapi-for-cs/src/proxy-ui-api/frontend/package.json#L15 has a working example of openapi definition bundling with `prepare-api-definition` (from `apidevtools/swagger-cli`), would it maybe be an easier route to take?

Reply:
I tried to avoid adding new npm packages here. Don't know if it's easier with the `swagger-cli` package but in the long run it might come in handy 👍 -> changed to suggested